test/async-hooks/test-httpparser.response.js

```diff
@@ -19,11 +19,8 @@ const hooks = initHooks();
 hooks.enable();
 
 const request = Buffer.from(
-  'HTTP/1.1 200 OK' + CRLF +
-  'Content-types: text/plain' + CRLF +
-  'Content-Length: 4' + CRLF +
-  CRLF +
-  'pong'
+  `HTTP/1.1 200 OK${CRLF}Content-types: text/plain${CRLF}` +
+  `Content-Length: 4${CRLF}${CRLF}pong`
```

Comment:
BTW That header is not mandatory. IMHO it should be removed.
Actually the only header that "MUST" be included is the `Status-Line` and two `CRLF`s
Ref: https://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html#sec6